### PR TITLE
Feature/bootstrap

### PR DESCRIPTION
**Pull request by semia**
Add bootstrap to views
Change bootstrap to bootstrap-sass
Add present_answers method to QuestionPresenter

### CODE REVIEW

File: spec/features/profile_spec.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   background do
     sign_in_user(user)
 
-    click_on 'My profile'
+    click_on "#{user.first_name} #{user.last_name}"
```
Comment (arnlen) [inline spec/features/profile_spec.rb]: Nice ! :)

File: spec/features/sign_in_spec.rb [hunk 2]
```diff
@@ -7,8 +7,7 @@
     let(:user) { build(:user) }
 
     scenario 'should sign in the user with valid credentials' do
-      expect(page).to have_content "Signed in successfully. Signed in as
-                                    #{user.first_name} #{user.last_name}"
+      expect(page).to have_content 'Signed in successfully.'
```
Comment (arnlen) [inline spec/features/sign_in_spec.rb]: Because we're now in FR, I guess this won't work anymore.

**Comment by arnlen** [comment]
capture d ecran 2016-06-09 a 22 08 09

😱 It's working!!! 😱

**Comment by arnlen** [comment]
@fabienalbi @semia : It's hard to review such view code, so I'll make general remark here:

- when there is something to show, show it! Place as much screenshots as you can in your PR. This will motivate reviewers to review your code, and it will allow much more interaction on your PR, since viewers won't have to checkout your branch locally. That's true for our internal projects, but that's particularly efficient for open source project contributions. :)
- try to summary the "Why" of your PR and not the "How". We often want to know what your PR is trying to do, before being interesting at the "how the magic happens thing". Even if it's useful for every PR, it's a requirement when it comes to the UI/UX PR, because the code is very hard to picture out without launching it in a browser.
- be careful about i18n: think to it when writing acceptance/integration tests, because depending of your languages (and the presence of absence of translations), your tests might fail sooner than expected.

That is said: nice work, the UI is much more usable like this, we'll be able to polish it a bit more later. ;)

**Comment by arnlen** [comment]
I've tried a rebase, but there are conflicts.
I let you rebase your branch on top on `develop` before merging it. ;)

**Comment by fabienalbi** [comment]
@arnlen : Thanks for the great advices about PR submission. This makes a lot of sense and be sure that we'll apply them on the following PRs.
As for the i18n part, it is definitely tricky, specs have been updated (i.e., translated) with the 'devise' PR that you just merged. Maybe we should rewrite them again so that they are not dependent on the language being used by the app. Food for thought for next week :)
The rank badge is too big on your screenshot, not sure what is going on. We'll look into it.
We'll take care of the rebase/merge on monday morning

**Comment by ManyBaby** [comment]
@arnlen I already learned a lot about the best practices just by reading those commentaries.  Thank you.

**Pull request merged**